src/components/Cart/Cart.js

```diff
@@ -0,0 +1,34 @@
+import { useTheme } from "../../contexts"
+import { getBgColor } from "../../utils"
+import { Main, Text } from "../Reusable"
+import { CartHeader, CartSection } from "./"
+
+const Cart = () => {
+    const { theme } = useTheme()
+
+    return (
+
+        <div
+            style={{
+                minHeight: '100vh'
+            }}
```

Comment:
suggestion - avoid use of inline styles

Reply:
actually, i wanted this to have priority over anything else. so gave this `min-height` as inline style.